src/components/Chat/ChatList/index.tsx

```diff
@@ -18,12 +21,32 @@ export default function ChatList({
   count,
 }: Props) {
   const navigate = useNavigate();
+
+  const profileList = roomImage
+    .map(({ name, imageUrl }) => {
+      return {
+        altValue: name,
+        imageUrl: imageUrl,
+      };
+    })
+    .slice(0, 3);
+  const profileLen = profileList.length;
+
+  const getProfileClassName = () => {
+    if (profileLen === 2) return 'double';
+    else if (profileLen === 3) return 'triple';
+    else return 'single';
```

Comment:
`return`에서 함수가 끝나기 때문에 `else`는 없어도 될 것 같습니다~
```suggestion
    if (profileLen === 3) return 'triple';
    return 'single';
```